modules/n1ql/pages/n1ql-language-reference/orderby.adoc

```diff
@@ -104,6 +104,8 @@ Where objects are the same size, objects are sorted in order of the first key in
 
 == Examples
 
+include::ROOT:partial$query-context.adoc[tag=section]
```

Comment:
Good shout, need to make a note to include this in the `capella` and `release/7.2` branches.